flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/expressions/RexNodeConverter.java

```diff
@@ -112,7 +113,7 @@ private RexNode visitScalarFunc(FunctionDefinition def, List<RexNode> child) {
 		} else if (BuiltInFunctionDefinitions.EQUALS.equals(def)) {
 			return relBuilder.call(SqlStdOperatorTable.EQUALS, child);
```

Comment:
maybe we should change all SqlStdOperatorTable to FlinkSqlOperatorTable?

Reply:
Yes, it is better to all use `FlinkSqlOperatorTable`.  Avoid next bug...